modules/howtos/examples/transactions-example.ts

```diff
@@ -129,38 +119,26 @@ async function main() {
         parameters: ['airline_137', 'AF'],
       })
 
-      // // Committing (the ctx.commit() call is optional)
-      // ctx.commit();
     })
   } catch (error) {
     if (error instanceof TransactionFailedError) {
-      // The operation failed. Both the monster and the player will be untouched.
-      //
-      // Situations that can cause this would include either the monster
-      // or player not existing (as get is used), or a persistent
-      // failure to be able to commit the transaction, for example on
-      // prolonged node failure.
+      console.error('Transaction did not reach commit point', error)
     }
 
     if (error instanceof TransactionCommitAmbiguousError) {
-      // Indicates the state of a transaction ended as ambiguous and may or
-      // may not have committed successfully.
-      //
-      // Situations that may cause this would include a network or node failure
-      // after the transactions operations completed and committed, but before the
-      // commit result was returned to the client
+      console.error('Transaction possibly committed', error)
     }
   }
   // end::examples[]
 
   // execute other exmaples
   try {
-    await get()
-    await getReadOwnWrites()
-    await replace()
-    await remove()
-    await insert()
-    await queryExamples()
+    await get() ; console.log("get…")
```

Comment:
will the log always be printed before the called function, or is that a race condition?

Reply:
I was just logging so I could see where the error had occurred.  there is no race.